app/src/main/java/com/example/intervaltimer/interval/IntervalItemTouchCallback.kt

```diff
@@ -45,6 +51,8 @@ class IntervalItemTouchCallback(
         val position = viewHolder.adapterPosition
         val intervalToDelete = cardAdapter.getItems()[position]
 
+        processTimeChanges(intervalToDelete, false)
```

Comment:
Why not just pass in `-intervalToDelete.length`, and remove the second parameter?

Reply:
The interval object itself needs to get passed, not just the length of the interval. This is because we need to persist it to the database, otherwise your suggestion would work fine.